### PR TITLE
Some changes triggered by #243

### PR DESCRIPTION
**Pull request by ArminJo**
The work of this day :-)
- Bumped version to 2.6
- Added compile flag STORE_CONFIGURATION_REPLY_IN_RAM.
- Added 2 bytes USB config data for Bootloader feature flags and application version.
- Added --info flag for micronucleus executable.
- Minor housekeeping.

### CODE REVIEW

File: commandline/library/micronucleus_lib.c [hunk 1]
```diff
@@ -97,6 +99,11 @@ micronucleus* micronucleus_connect(int fast_mode) {
 
           assert(res >= 6);
```
Comment (cpldcpu) [inline commandline/library/micronucleus_lib.c:100]: This assert should also be adjusted if the descriptor length is increased.

File: firmware/configuration/t88_default/bootloaderconfig.h [hunk 2]
```diff
@@ -24,16 +23,16 @@
...
 
 #define USB_CFG_IOPORTNAME      D
-  /* This is the port where the USB bus is connected. When you configure it to
-   * "B", the registers PORTB, PINB and DDRB will be used.
-   */
+/* This is the port where the USB bus is connected. When you configure it to
+ * "B", the registers PORTB, PINB and DDRB will be used.
+ */
 
-#define USB_CFG_DMINUS_BIT      2
```
Comment (cpldcpu) [inline firmware/configuration/t88_default/bootloaderconfig.h:31]: why is it changed?

**Review by cpldcpu — APPROVED** [review]

**Comment by cpldcpu** [comment]
Nice!

File: commandline/library/micronucleus_lib.c (hunk 1 above)
Comment (ArminJo) [reply commandline/library/micronucleus_lib.c:100]: It must be at 6 to be compatible with older bootloaders sending only 6 bytes.

File: firmware/configuration/t88_default/bootloaderconfig.h (hunk 2 above)
Comment (ArminJo) [reply firmware/configuration/t88_default/bootloaderconfig.h:31]: Armin	10.03.2021	Swapped D+ and D- for t88 to support MH-ET LIVE Tiny88 boards	#define USB_CFG_DMINUS_BIT      1**

File: commandline/library/micronucleus_lib.c (hunk 1 above)
Comment (cpldcpu) [reply commandline/library/micronucleus_lib.c:100]: ah ok

**Comment by cpldcpu** [comment]
Are you also planning to bump the version number in the readme?

**Comment by ArminJo** [comment]
very good idea 😀

**Comment by cpldcpu** [comment]
Shall I tag it for release or do you want to wait if there are some issues?

**Comment by ArminJo** [comment]
I tested it carefully (but only with t85) so I think I would be a good idea if you tag it now.
In turn I would like to inform Spence Konde about the changes.

**Comment by cpldcpu** [comment]
ok, done.

I noticed it can be done easily in the web interface :) I used to do it with the commandline in the past.

**Comment by ArminJo** [comment]
Fine 👍 
Never tested the web interface or do you mean the browser pages? I use SmartGit for most git processing.

**Comment by ArminJo** [comment]
I also struggled a few hours with my version of CI/CD commiting the binaries back to the repo...
Can't you not just retag the master-latest pre release and convert it to a release?

**Comment by cpldcpu** [comment]
Not sure this is possible, because what is tagged is actually not the release, but the repository.

**Comment by cpldcpu** [comment]
oki, done.

**Pull request merged**